gaia-fast-list.js

```diff
@@ -894,6 +913,7 @@ Internal.prototype = {
   cacheImage(image, index) {
     if (!this.el.imageCacheLength) return;
     if (!this.el.imageCacheSize) return;
+    if (this.images.hash[index]) return;
```

Comment:
This guards against us very rarely attempting to cache the same image twice. Was causing intermittent test failures.